environments/ncovtools.yml

```diff
@@ -0,0 +1,24 @@
+channels:
+  - conda-forge
+  - bioconda
+  - defaults
+dependencies:
+  - snakemake
+  - mafft
+  - augur
+  - pysam
+  - r-base=3.6.1
+  - iqtree<2
+  - bioconductor-ggtree=2.0.0
```

Comment:
can you add r-argparse ? Just was added yesterday

Reply:
Added